### PR TITLE
JAPI-384 Provide ECL to generate sample Iris dataset

### PR DESCRIPTION
**Pull request by rpastrana**
Signed-off-by: Rodrigo Pastrana <Rodrigo.Pastrana@lexisnexisrisk.com>

### CODE REVIEW

**Comment by rpastrana** [comment]
@jpmcmu @JamesDeFabia Please review.
Jim, I've never delivered ECL, do we need a header? Did I break any norms?

File: Examples/src/main/ecl/IrisDS.ecl [hunk 1]
```diff
@@ -0,0 +1,163 @@
+dsRecord := RECORD
```
Comment (JamesDeFabia) [inline Examples/src/main/ecl/IrisDS.ecl:1]: In other ECL Examples we have a comment block with copyright. I am not sure if needed

File: Examples/src/main/ecl/IrisDS.ecl [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+  real a1 ;
```
Comment (JamesDeFabia) [inline Examples/src/main/ecl/IrisDS.ecl:2]: Our style in ECL is to uppercase keywords, so I would:
dsRecord := RECORD
      REAL a1 ;
      REAL a2 ;
      REAL a3 ;
      REAL a4 ;
      REAL class;
    END;

File: Examples/src/main/ecl/IrisDS.ecl [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+{6.7,3.3,5.7,2.5,2},
+{6.7,3.0,5.2,2.3,2},
+{6.3,2.5,5.0,1.9,2},
+{6.5,3.0,5.2,2.0,2},
+{6.2,3.4,5.4,2.3,2},
+{5.9,3.0,5.1,1.8,2}], dsRecord);
+
+dsname := '~samples::iris_sample';
+
+OUTPUT(irisDS,,dsname,overwrite);
```
Comment (JamesDeFabia) [inline Examples/src/main/ecl/IrisDS.ecl:163]: Uppercase OVERWRITE.

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
A couple of comments inline

**Comment by rpastrana** [comment]
@JamesDeFabia please review latest commit

**Review by JamesDeFabia — APPROVED** [review]
Looks good from my POV

**Review by jpmcmu — APPROVED** [review]
@rpastrana looks good

**Pull request merged**